code/__DEFINES/dcs/signals.dm

```diff
@@ -979,3 +979,6 @@
 #define COMSIG_HUMAN_EARLY_UNARMED_ATTACK "human_early_unarmed_attack"
 ///from mob/living/carbon/human/UnarmedAttack(): (atom/target, proximity)
 #define COMSIG_HUMAN_MELEE_UNARMED_ATTACK "human_melee_unarmed_attack"
+
+///from the shuttle subsystem, occurs when the shuttle is stranded
+#define COMSIG_SHUTTLE_STRANDED "shuttle_stranded"
```

Comment:
Instead of a stranded signal, how about changing the shuttle mode through a setter and sending a signal when it changes?
That would be a more generic approach.

Reply:
Sure, I'll take a look.